back/spec/acceptance/users_spec.rb

```diff
@@ -525,465 +545,578 @@
           end
         end
       end
-    end
 
-    get 'web_api/v1/users' do
-      with_options scope: :page do
-        parameter :number, 'Page number'
-        parameter :size, 'Number of users per page'
-      end
-      example 'Get all users as non-admin', document: false do
-        do_request
-        assert_status 401
+      get 'web_api/v1/users/by_slug/:slug' do
+        let(:user) { create :user }
+        let(:slug) { user.slug }
+
+        example_request 'Get one user by slug includes user block data' do
+          expect(status).to eq 200
+          json_response = json_parse response_body
+          expect(json_response.dig(:data, :attributes)).to have_key(:blocked)
+          expect(json_response.dig(:data, :attributes)).to have_key(:block_start_at)
+          expect(json_response.dig(:data, :attributes)).to have_key(:block_end_at)
+          expect(json_response.dig(:data, :attributes)).to have_key(:block_reason)
+        end
```

Comment:
This is a new test (I refactored the file, so may not be obvious which tests are new)